cmd/kubeadm/app/phases/controlplane/manifests_test.go

```diff
@@ -767,6 +767,95 @@ func TestGetControllerManagerCommand(t *testing.T) {
 				"--node-cidr-mask-size=80",
 			},
 		},
+		{
+			name: "custom certs dir",
```

Comment:
This test case is enough for me, the rest of the `v1.11.3` in TestGetControllerManagerCommand are not necessary (since we already have them for 1.12)